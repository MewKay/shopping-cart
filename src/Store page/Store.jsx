import { Link } from "react-router-dom";

const Store = function createStoreComponent() {
  return (
    <div>
      <h2>View Our Products</h2>
      <div>
        <Link to={"men's clothing"}>Men&apos;s Clothing</Link>
        <Link to={"women's clothing"}>Women&apos;s Clothing</Link>
        <Link to={"electronics"}>Electronics</Link>
        <Link to={"jewelery"}>Jewelery</Link>
      </div>
      <Link to={"all"}>View All</Link>
    </div>
  );
};

export default Store;
