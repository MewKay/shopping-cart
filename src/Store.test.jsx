import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { describe, expect, it } from "vitest";
import Store from "./Store";

describe("Store component", () => {
  it("renders correctly", () => {
    const { container } = render(<Store />, { wrapper: BrowserRouter });

    expect(container).toMatchSnapshot();
  });
});
