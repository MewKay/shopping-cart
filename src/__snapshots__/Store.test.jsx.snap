// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Store component > renders correctly 1`] = `
<div>
  <div>
    <h2>
      View Our Products
    </h2>
    <div>
      <a
        href="/men's clothing"
      >
        Men's Clothing
      </a>
      <a
        href="/women's clothing"
      >
        Women's Clothing
      </a>
      <a
        href="/electronics"
      >
        Electronics
      </a>
      <a
        href="/jewelery"
      >
        Jewelery
      </a>
    </div>
    <a
      href="/all"
    >
      View All
    </a>
  </div>
</div>
`;
